readme.md

```diff
@@ -363,13 +363,15 @@ test(t => {
 });
 ```
 
-You can also use your own local Babel version:
+To use your own local Babel version, specify the version you want in your package.json, and ensure you're using npm@3 (or run `$ npm dedupe`), so the dependency will be deduped.
+
+*We don't yet [support Babel 6](https://github.com/sindresorhus/ava/pull/221), but you can still use it in your own project.*
 
 ```json
 {
 	"devDependencies": {
 		"ava": "^0.6.0",
-		"babel-core": "^5.8.0"
+		"babel-core": "5.8.11"
```

Comment:
Either bump this to show `^6`, or move this line

> We don't yet support Babel 6...

Down below the example code and preface it with `Note:`

> ```js
> {
>   "devDependencies": {
>    ...
> } 
> ```
>   
> *Note: We don't yet support Babel 6, but you can still use it in your own project*

I personally think it would be better to just give the Babel 6 example. In essence, all we are doing right now is explaining to people how semver and dedupe works. I think it undersells the best part of this change: that you can now use Babel 6.